appsdk/sdk.go

```diff
@@ -106,33 +127,32 @@ func (sdk *AppFunctionsSDK) HTTPPost(url string, mimeType string) func(*appconte
 	return transforms.HTTPPost
 }
 
-// HTTPPostJSON will add an export function that sends data from the previous function to the specified Endpoint via http POST with a mime type of application/json.
+// HTTPPostJSON sends data from the previous function to the specified Endpoint via http POST with a mime type of application/json.
```

Comment:
Doesn't have to be a previous function, correct?
Change to "sends input data to the specified"